front/tests/006-stdcm-page.spec.ts

```diff
@@ -25,9 +27,9 @@ test.describe('STDCM page', () => {
     // Opens the scenario explorator and selects project, study and scenario
     await playwrightSTDCMPage.openScenarioExplorator();
     await playwrightSTDCMPage.getScenarioExploratorModalOpen();
-    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration project');
-    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration study');
-    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration scenario');
+    await playwrightSTDCMPage.clickItemScenarioExploratorByName(project.name);
```

Comment:
Can you keep the previous name here ? This way, we are sure the project/study/scenario have not been deleted/edited by another test